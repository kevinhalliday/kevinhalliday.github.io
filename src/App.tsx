/**
 * Define top level App component.
 */

import React from 'react'
import { hot } from 'react-hot-loader/root'
import { Switch, BrowserRouter as Router, Route } from 'react-router-dom'

import Home from 'pages/Home'

const App = () => (
  <Router>
    <Switch>
      <Route path="/">
        <Home />
      </Route>
    </Switch>
  </Router>
)

export default hot(App)
