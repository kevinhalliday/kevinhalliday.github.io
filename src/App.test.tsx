/**
 * Top level application tests.
 */

import React from 'react'
import { render } from '@testing-library/react'

import App from './App'

describe('App', () => {
  test('matches snapshot', () => {
    const app = render(<App />)
    expect(app).toMatchSnapshot()
  })
})
