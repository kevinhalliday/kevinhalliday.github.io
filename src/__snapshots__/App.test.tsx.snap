// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="page"
        id="home"
      >
        <div
          class="self-portrait"
        >
          <div
            class="frame"
          >
            <div
              class="head"
            >
              <div
                class="hair"
              >
                <div
                  class="strand"
                />
                <div
                  class="strand"
                />
                <div
                  class="strand"
                />
                <div
                  class="strand"
                />
              </div>
              <div
                class="eyes"
              >
                <div
                  class="eye"
                >
                  <div
                    class="brow"
                  />
                  <div
                    class="white"
                    style="z-index: 0; top: 0px; left: 0px; opacity: 1; transform: scale(1);"
                  >
                    <div
                      class="iris"
                      style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                    >
                      <div
                        class="pupil"
                        style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="eye"
                >
                  <div
                    class="brow"
                  />
                  <div
                    class="white"
                    style="z-index: 0; top: 0px; left: 0px; opacity: 1; transform: scale(1);"
                  >
                    <div
                      class="iris"
                      style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                    >
                      <div
                        class="pupil"
                        style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                      />
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="nose"
              />
              <div
                class="mouth"
                style="bottom: 10%; height: 10%; border-radius: 0% 0% 100% 100%;"
              >
                <div
                  class="teeth"
                />
                <div
                  class="tongue"
                />
              </div>
            </div>
            <div
              class="shirt"
            />
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="page"
      id="home"
    >
      <div
        class="self-portrait"
      >
        <div
          class="frame"
        >
          <div
            class="head"
          >
            <div
              class="hair"
            >
              <div
                class="strand"
              />
              <div
                class="strand"
              />
              <div
                class="strand"
              />
              <div
                class="strand"
              />
            </div>
            <div
              class="eyes"
            >
              <div
                class="eye"
              >
                <div
                  class="brow"
                />
                <div
                  class="white"
                  style="z-index: 0; top: 0px; left: 0px; opacity: 1; transform: scale(1);"
                >
                  <div
                    class="iris"
                    style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                  >
                    <div
                      class="pupil"
                      style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                    />
                  </div>
                </div>
              </div>
              <div
                class="eye"
              >
                <div
                  class="brow"
                />
                <div
                  class="white"
                  style="z-index: 0; top: 0px; left: 0px; opacity: 1; transform: scale(1);"
                >
                  <div
                    class="iris"
                    style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                  >
                    <div
                      class="pupil"
                      style="transform: translate3d(calc(0px - 50%),calc(0px - 50%),0);"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              class="nose"
            />
            <div
              class="mouth"
              style="bottom: 10%; height: 10%; border-radius: 0% 0% 100% 100%;"
            >
              <div
                class="teeth"
              />
              <div
                class="tongue"
              />
            </div>
          </div>
          <div
            class="shirt"
          />
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
